<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="selPesoLE">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: #E6ECEE;
	border: 0px;
	border-radius: 10px;
	font: 12pt &quot;Poppins&quot;;
}

QLineEdit:hover{
	border: 1px solid black;
}</string>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Seleccionar peso</string>
    </property>
   </widget>
   <widget class="QPushButton" name="taraBtn">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #D9ECF2;
	border: 0px;
	border-radius: 10px;
	font: 12pt &quot;Poppins&quot;;
}

QPushButton:hover{
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Set tara</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciarBtn">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #D9ECF2;
	border: 0px;
	border-radius: 10px;
	font: 12pt &quot;Poppins&quot;;
}

QPushButton:hover{
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detenerBtn">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>230</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: #D9ECF2;
	border: 0px;
	border-radius: 10px;
	font: 12pt &quot;Poppins&quot;;
}

QPushButton:hover{
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>Detener</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>350</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poppins&quot;;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%p%</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #E6ECEE;
border: 0px;
border-bottom-left-radius: 30px;
border-bottom-right-radius: 30px;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Poppins&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Panel de control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="alertaFrame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #E6ECEE;
border: 2px solid black;
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="alertaBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: #D9ECF2;
	border: 0px;
	border-radius: 10px;
	font: 12pt &quot;Poppins&quot;;
}

QPushButton:hover{
	border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
    <widget class="QLabel" name="alertaLbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Poppins&quot;;
border: 0px;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Panel de control&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="statusIcon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Poppins&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>457</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>fondo</zorder>
   <zorder>selPesoLE</zorder>
   <zorder>taraBtn</zorder>
   <zorder>iniciarBtn</zorder>
   <zorder>detenerBtn</zorder>
   <zorder>progressBar</zorder>
   <zorder>widget</zorder>
   <zorder>alertaFrame</zorder>
   <zorder>statusIcon</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
